docs/docs-components/Header.js

```diff
@@ -63,12 +63,24 @@ function Header() {
   );
 
   useEffect(() => {
+    const devModeSetFromUrl = router.query.devexample && router.query.devexample.length > 0;
+
+    // do not show switch if set via url
     if (
+      devModeSetFromUrl &&
+      (router.query.devexample === 'true' || router.query.devexample === 'false')
```

Comment:
I'm curious why we want to hide if `?devexample=false`?

Reply:
Because it still overrides localhost

Reply:
Oh, whoops, nvm, it doesn't. I thought it was a hard boolean here. I guess it doesn't matter, I can remove